src/main/java/hudson/plugins/promoted_builds/Promotion.java

```diff
@@ -159,7 +180,7 @@ public String getUserName(){
         }
         return values;
       }
-      
+
```

Comment:
Even whitespace changes can cause a merge conflict, so we commonly try to avoid them in order to simplify the work of plugin maintainers and other committers